src/Razor/test/Microsoft.VisualStudio.Razor.IntegrationTests/RenameTests.cs

```diff
@@ -68,16 +64,20 @@ public async Task Rename_ComponentAttribute_FromCSharpInRazor()
     public async Task Rename_ComponentAttribute_FromCSharpInCSharp()
     {
         // Create the file
+        const string MyComponentRazorPath = "MyComponent.razor";
         await TestServices.SolutionExplorer.AddFileAsync(RazorProjectConstants.BlazorProjectName,
-            "MyComponent.razor",
+            MyComponentRazorPath,
             """
                 @MyProperty
                 """,
-            open: false,
+            open: true,
```

Comment:
Either opened or closed should work, but we're curious about rename rather than our project system so I've moved a copy of this into the "Synchronization" tests and tried to get this in line.

Reply:
~I don't like this... "we're curious about rename", yes, but rename of closed files is a valid scenario that could break for its own special reasons, and should be verified. Maybe this property could be parameterized?~ Nevermind, you're closing it again, so all good. Perhaps a comment that its deliberate?